diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 5d5dfe4..f66c49f 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -45,10 +45,12 @@ if(${RNS_NEW_ARCH_ENABLED})
         ReactAndroid::jsi
         fbjni::fbjni
         android
+        c++_shared
     )
 else()
     target_link_libraries(rnscreens
         ReactAndroid::jsi
         android
+        c++_shared
     )
 endif()
